scripts/fetch-asyncapi-example.js

```diff
@@ -86,7 +86,7 @@ const buildCLIListFromExamples = async () => {
     }
   });
 
-  const exampleList = await Promise.all(buildExampleList);
+  const exampleList = (await Promise.all(buildExampleList)).filter(item => item!==null);
```

Comment:
Nitpick: 
```suggestion
  const exampleList = (await Promise.all(buildExampleList)).filter(item => item !== null);
```

It seems we are not passing any linter to the scripts under `scripts` dir (I don't know the reason cc @Souvikns). Anyway this nitpick I suggest improves readability and needs to be considered.